test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterMovementReferenceNumberControllerSpec.scala

```diff
@@ -273,8 +273,10 @@ class EnterMovementReferenceNumberControllerSpec
         }
         val result = performAction(JourneyBindable.Single, enterMovementReferenceNumberKey -> entryNumber.value)
 
-        status(result)                 shouldBe 303
-        redirectLocation(result).value shouldBe "/claim-for-reimbursement-of-import-duties/enter-declaration-details"
+        status(result) shouldBe 303
+        redirectLocation(
+          result
+        ).value        shouldBe "/claim-for-reimbursement-of-import-duties/single/enter-declaration-details"
```

Comment:
Can you refactor there using reverse routes please?
Like in all other controller tests

Reply:
@gabor-dorcsinecz that's not really a proper test then as you are asserting implementation against implementation. What Sergey has there is correct. Test implementation asserted against expected value.

Reply:
@mikail-khan are we good to merge?